packages/safe-tools-client/app/components/schedule-payment-form-action-card/index.gts

```diff
@@ -272,6 +278,27 @@ export default class SchedulePaymentFormActionCard extends Component<Signature>
     return state;
   });
 
+  @use usdToken = resource(() => {
+    const state: ConfiguredUsdTokenState = new TrackedObject({
+      isLoading: true,
+    });
+    let isWalletConnected = this.wallet.isConnected;
+    if (!isWalletConnected) {
```

Comment:
Looks like the variable isn't really needed and this could be reduced to: `if (!this.wallet.isConnected) { ... }`